sparkbox/eslint-config-sparkbox
# Move devDependencies that end-users need installed to dependencies

Corresponding issue: #25

**Testing instructions**
1. Confirm that the change matches what is expected in [the issue](https://github.com/sparkbox/eslint-config-sparkbox/issues/25).
2. Pull down this branch.
3. Reinstall all the dependencies: `rm -rf node_modules && npm install`
4. Run tests, and confirm that they pass: `npm test`
